web/pdf_page_view.js

```diff
@@ -564,17 +575,38 @@ class PDFPageView {
       }
     }
 
+    const postponeDrawing = postponeDrawingDelay >= 0;
+
+    if (postponeDrawing) {
+      this.textLayer?.hide();
+    } else {
+      this.textLayer?.show();
+    }
```

Comment:
*Sorry, but I'm not following:* Why are we toggling the textLayer visibility here, since that's really not immediately clear to me?

If we actually need to do this, is there perhaps a slightly more appropriate spot to do this (e.g. in the `cancelRendering` method)?

Reply:
The idea is to hide the text layer during the zoom session in order to not have to reflow it (which can take some time).
Here are some pictures from the profiler:
(page 1 of tracemonkey without hiding the text layer)
![image](https://user-images.githubusercontent.com/5641725/208422981-69fb4567-6a47-4f5d-af45-ce467a504482.png)
(the same but in hiding the layer)
![image](https://user-images.githubusercontent.com/5641725/208423153-67d7a631-aae0-499f-9687-009b7cf1de5a.png)

Reply:
After thinking about that, I'm not sure it's a good idea to move that stuff in `cancelRendering` because we don't want to cancel something here and it doesn't make to hide it in the non-postpone case.